### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -8,6 +8,9 @@ hipBLASLt
 hipBLASLtExt
 =============
 
+2023-08-10
+----------
```
Comment (jichangjichang) [inline docs/changelog.rst:12]: Changed feature follow by RCOM version but not by date

File: docs/changelog.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+*****************************
+hipBLASLt Changelog
+*****************************
+
+hipBLASLt
+=============
+
```
Comment (jichangjichang) [inline docs/changelog.rst:7]: Could you help me the add that hipblasDatatyp_t has been deprecated and now we use hipblasltDatatype_t?

File: docs/changelog.rst (hunk 1 above)
Comment (KKyang) [reply docs/changelog.rst:12]: Which version? 5.7?

File: docs/changelog.rst (hunk 2 above)
Comment (KKyang) [reply docs/changelog.rst:7]: Ok

**Review by jichangjichang — APPROVED** [review]
LGTM

**Pull request merged**